source/making_payments/index.html.md.erb

```diff
@@ -305,7 +305,7 @@ that’s not `null`, you can cancel the payment. For example:
 
 ```
 "cancel": {
-  "href": "https://publicapi.payments.service.gov.uk/v1/payments/{paymentId}/cancel",
+  "href": "https://publicapi.payments.service.gov.uk/v1/payments/{PAYMENT_ID}/cancel",
```

Comment:
Possibly out of scope, but I wonder if this example should have an actual example of a payment id (eg `hu20sqlact5260q2nanm0q8u93`) rather than the placeholder? Because a placeholder suggests something the user needs to replace. That would be consistent with other response examples further up the page.